cordova-plugin-one/README.md

```diff
@@ -10,9 +10,12 @@ $ cordova plugin add cordova-plugin-one
 ```
```

Comment:
Ideally update the whole structure to match order of native methods so it's easier to review if you look at one or the other. Also please add any requirements at the top similar to the native SDKs.